runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaRunner.java

```diff
@@ -81,12 +82,16 @@ private SamzaRunner(SamzaPipelineOptions options) {
   }
 
   public PortablePipelineResult runPortablePipeline(RunnerApi.Pipeline pipeline, JobInfo jobInfo) {
-    final String dotGraph = PipelineDotRenderer.toDotString(pipeline);
-    LOG.info("Portable pipeline to run:\n{}", dotGraph);
+    Optional<String> jsonGraphOpt = PipelineJsonRenderer.toJsonString(pipeline);
+    String jsonGraph = "";
+    if (jsonGraphOpt.isPresent()) {
+      jsonGraph = jsonGraphOpt.get();
+    }
```

Comment:
I have to admit that changing `PipelineJsonRenderer.toJsonString` to return Optional actually made it less readable, let's revert it back to original.